src/wp-includes/class-wp-scripts.php

```diff
@@ -875,6 +876,18 @@ private function get_intended_strategy( $handle ) {
 		return isset( $script_args['strategy'] ) ? $script_args['strategy'] : false;
 	}
 
+	/**
+	 * Check if the strategy passed is one of the delayed strategies.
+	 *
+	 * @param string $strategy The strategy to check.
+	 * @return bool True if $strategy is one of the delayed strategy.
+	 */
+	private function is_delayed_strategy( $strategy ) {
```

Comment:
for core, convention is prefixing function names with `_`

Reply:
@adamsilverstein I don't think so? Private methods should never be prefixed, and even for privately annotated functions there is no clear convention. When I started contributing to core years ago, I got feedback from several seasoned core team members that I should only put `@access private` on the function, but not underscore-prefix it. Opinions appear to differ on that, but at least for class methods we should not use prefixes.

Reply:
For global functions, we use the underscore. For private class methods, I believe @felixarntz is correct.

Also: naming things is hard. What about `is_non_blocking_strategy`? Seems that is generally the intent of that method anyway, right?

Reply:
@joemcgill that name looks perfect to me. I'll rename it.